README.md

```diff
@@ -45,12 +45,22 @@ The function `expectMessage(value, message, style = {})` takes three arguments:
 
 #### Example
 
+With pre-defined style:
 ```js
 test("simple test", () => {
-  expectMessage(1 + 1, "This is a level 3 message", 3).toBe(11);
+  expectMessage(1 + 1, "This is an error message", style="error").toBe(11);
```

Comment:
🤔 this does not look like javascript, is it `{style: "error"}`?

Reply:
Yes ! I've followed the doc blindly 😅

But it worked, so I did not asked myself if it was right or not. 


<img width="658" alt="Capture d’écran 2021-01-27 à 09 20 36" src="https://user-images.githubusercontent.com/34440340/105963149-efc9eb00-6080-11eb-9c4e-7dc119de6e35.png">